spec.md

```diff
@@ -26,6 +27,29 @@ Enter CloudEvents, a specification for describing event data in a common way.
 CloudEvents seeks to ease event declaration and delivery across services,
 platforms and beyond.
 
+# Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to
+be loosely coupled during development, deployed independently, and later
+can be connected to create new applications.
+
+The goal of the CloudEvents specification is to define interoperability of event
+systems that allow services to produce or consume events, where the producer and
+consumer can be developed and deployed independently.  A producer MUST be able
+to generate events before a consumer is listening, and a consumer MUST be able
```

Comment:
I'd switch these to "needs to" because this isn't technically part of the normative section of the spec. These don't place any formal requirements on people implementing the spec.